documentation/modules/proc-installing-crds.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-install-crds-{context}']
```

Comment:
I think we should not name it `proc-install-crds` as it does lot more than just installing CRDs. So the section ID is a bit confusing.